kong/conf_loader/init.lua

```diff
@@ -949,10 +949,10 @@ local function load(path, custom_conf, opts)
     end
   end
 
-  -- TODO: remove this when cluster_rpc is ready for GA
-  if conf.cluster_rpc then
-    log.warn("Cluster RPC has been forcibly disabled")
-    conf.cluster_rpc = "off"
+  -- TODO: remove this when incremental sync is ready for GA
```

Comment:
is this comment correct? If incremental sync requires `cluster_rpc`, that will be the case even when the feature is GA, right?

Reply:
Thank you, I will remove this line.